### DISCUSSION TITLE
What should I do after I graduate from junior high school and the senior high school does not issue a student ID?

### DISCUSSION THREAD

**Discussion by Ark-Aak** [community/community#143361] [GitHub Education] [answered]
### Hi everyone,

Because I graduated, my student ID has expired, but the new school has not issued a new student ID.

I tried to apply using my diploma but was rejected.

My school does not have an education mailbox or transcript.

I have an admission letter, but there is no student ID number and date on it.

Are there any other documents that can prove my identity?

**Comment by ravindudil5han** [comment, marked as answer]
If you’re trying to prove your student status without an active student ID, there are several alternative documents you might consider using. Here’s a straightforward approach:

1. **Request an Enrollment Verification Letter**: Many schools can provide an official enrollment verification letter that confirms you’re currently a student. This letter usually includes your name, the program or course you’re enrolled in, and sometimes your expected graduation date. Contact your school’s registrar or student services office for this document.

2. **Use Your Admission Letter Alongside Additional Documentation**: If your admission letter doesn’t have all the details needed, try combining it with other documents, such as:
   - A **payment receipt** or **tuition invoice** showing recent payment for classes.
   - Any **orientation or welcome emails** that mention your enrollment or provide details about your program.
   - A **course registration confirmation** for the current or upcoming semester.

3. **Consider a School Contact Reference**: Some applications allow you to list a contact at your school (like a registrar, advisor, or professor) who can confirm your enrollment. If this is an option, you could ask someone in your school’s administration if they’d be willing to verify your status if needed.

4. **Explore a Temporary Student ID or Pass**: Some schools issue temporary ID cards, especially if there’s a delay in processing new student IDs. Check with student services to see if this is available to you.

Each institution may have different requirements, so it’s best to reach out directly to them to confirm what alternative documents they will accept.
